saleor_gql_loader/data_loader.py

```diff
@@ -65,6 +65,117 @@ def __init__(self, auth_token, endpoint_url="http://localhost:8000/graphql/"):
         self.headers = {"Authorization": "Bearer {}".format(auth_token)}
         self.endpoint_url = endpoint_url
 
+    def update_shop_settings(self, **kwargs):
```

Comment:
the id of the shop is required ? If so we should make it mandatory as a separate argument of the function

Reply:
Nope, there is just one shop supported in saleor for now, no ID required

I also checked for mandatory fields or default suggestions but didn't find any

Reply:
ok perfect then